examples/test/model/index.html

```diff
@@ -11,13 +11,16 @@
       <a-assets>
```

Comment:
In the latest and greatest master, that warning has been eliminated by using RawShaderMaterial for the sdf and msdf text shaders.